modules/charon-core/src/main/java/org/wso2/charon3/core/objects/User.java

```diff
@@ -915,6 +917,96 @@ private void setRole(ComplexAttribute groupPropertiesAttribute) throws CharonExc
         }
     }
 
+    /**
+     * Set the assigned V2 roles of the user.
+     *
+     * @param role RoleV2 object.
+     * @throws CharonException     CharonException.
+     * @throws BadRequestException BadRequestException.
+     */
+    public void setRoleV2(RoleV2 role) throws CharonException, BadRequestException {
+
+        SimpleAttribute valueSimpleAttribute = null;
+        SimpleAttribute displaySimpleAttribute;
```

Comment:
no usage

Reply:
same for the below ones

Reply:
fixed in [25b26c2](https://github.com/wso2/charon/pull/392/commits/25b26c2228a54f30201b37c15e189987e55ccf32)